[CI] XPackRestIT test {yaml=snapshot/10_basic/Create a source only snapshot and then restore it} failing

**Build Scans:**
- [elasticsearch-intake #39547 / part1](https://gradle-enterprise.elastic.co/s/azzduelwq5lco)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:yamlRestTest" --tests "org.elasticsearch.xpack.test.rest.XPackRestIT.test {yaml=snapshot/10_basic/Create a source only snapshot and then restore it}" -Dtests.seed=B8ABA084C9B409A0 -Dtests.locale=en-BZ -Dtests.timezone=Etc/GMT-9 -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.test.rest.XPackRestIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Byaml%3Dsnapshot%2F10_basic%2FCreate%20a%20source%20only%20snapshot%20and%20then%20restore%20it%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: org.junit.internal.runners.model.MultipleFailureException: There were 11 errors:
  java.lang.AssertionError(field [snapshot.shards.successful] does not match the expected value
Expected: <1>
     but: was <3>)
  java.lang.AssertionError(expected [2xx] status code but api [snapshot.restore] returned [500 Internal Server Error] [{"error":{"root_cause":[{"type":"snapshot_restore_exception","reason":"[default:test_repo_restore_1:test_snapshot/q24f6rwQSEmfdPbzvVqI7g] cannot restore index [.ml-annotations-000001] because an open index with same name already exists in the cluster. Either close or delete the existing index or restore the index under a different name by providing a rename pattern and replacement name","stack_trace":"org.elasticsearch.snapshots.SnapshotRestoreException: [default:test_repo_restore_1:test_snapshot/q24f6rwQSEmfdPbzvVqI7g] cannot restore index [.ml-annotations-000001] because an open index with same name already exists in the cluster. Eithe
[truncated]
```

**Issue Reasons:**
- [main] 1 consecutive failures in test test {yaml=snapshot/10_basic/Create a source only snapshot and then restore it}
- [main] 1 consecutive failures in step part1
- [main] 1 consecutive failures in pipeline elasticsearch-intake

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -545,9 +545,6 @@ tests:
 - class: org.elasticsearch.reindex.management.ReindexRethrottleRelocationIT
   method: testRethrottleByOriginalTaskIdAfterRelocation
   issue: https://github.com/elastic/elasticsearch/issues/148893
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {yaml=snapshot/10_basic/Create a source only snapshot and then restore it}
-  issue: https://github.com/elastic/elasticsearch/issues/148917
 - class: org.elasticsearch.xpack.ccr.index.engine.FollowingEngineTests
   method: testProcessOnceOnPrimary
   issue: https://github.com/elastic/elasticsearch/issues/148943

--- a/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/snapshot/10_basic.yml
+++ b/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/snapshot/10_basic.yml
@@ -42,7 +42,10 @@ setup:
         snapshot: test_snapshot
         wait_for_completion: true
         body: |
-          { "indices": "test_index" }
+          {
+            "indices": "test_index",
+            "include_global_state": false
+          }
 
   - match: { snapshot.snapshot: test_snapshot }
   - match: { snapshot.state : SUCCESS }

